lib/Transforms/ConvertIfToSelect/ConvertIfToSelect.cpp

```diff
@@ -0,0 +1,72 @@
+#include "lib/Transforms/ConvertIfToSelect/ConvertIfToSelect.h"
+
+#include "mlir/include/mlir/Dialect/SCF/IR/SCF.h"  // from @llvm-project
+#include "mlir/include/mlir/IR/Operation.h"        // from @llvm-project
+#include "mlir/include/mlir/IR/PatternMatch.h"     // from @llvm-project
+#include "mlir/include/mlir/Transforms/GreedyPatternRewriteDriver.h"  // from @llvm-project
+#include "mlir/include/mlir/Transforms/Passes.h"  // from @llvm-project
+
+using namespace mlir;
+using namespace mlir::scf;
```

Comment:
While there's sometimes good reasons to use `using namespace ...`, I don't think it's necessary here.
The first one isn't necessary anyway, as everything is already happening in `namespace mlir {...}`.
For the second one, I think it'd actually be clearer to write `scf::IfOp` instead of `IfOp` to make it clearer what exact op you're targeting.